test/DynamoCoreTests/UnitsOfMeasureTests.cs

```diff
@@ -1045,7 +1044,7 @@ public void ParseExpressionIncompatible()
         {
             Assert.That(() => { DynamoUnits.Utilities.ParseExpression("3mm"); }, Throws.Exception);
         }
-        [Test, Category("UnitTests")]
+        [Test, Category("UnitTests"), Category("FailureNET6")]
```

Comment:
why does this fail for net6?

Reply:
there is an access violation in forge units - I think @pinzart90 already found it but I'm not sure if theres a task to fix it. @pinzart90 can you update?